src/components/App/App.scss

```diff
@@ -33,7 +33,8 @@
   display: flex;
   position: fixed;
   z-index: 12;
-  left: 6%; top: 4%; right: 6%;
+  left: 20px; top: 20px;

```

Comment:
I would do `top: 15px;` that way we won't have to go through and adjust the workspace height/top to match the search bar.

Reply:
I wouldn't want to go less than 20px. The wireframes are actually looking for 32px but that would intersect with the workspace so I decided to go with 20px. I don't think we need to change the workspace height/top, but please compare to the deployed site and let me know if any views are not looking good.